examples/preprocessing/otp.py

```diff
@@ -35,7 +35,7 @@
 dipole_number = 1
 data_path = bst_phantom_elekta.data_path()
 raw = read_raw_fif(
-    op.join(data_path, 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif'))
+    (data_path / 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif'))
```

Comment:
```suggestion
    data_path / 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif')
```
remove extra parentheses

Reply:
The parenthesis is required for the 'read_raw_fif' import right?

Reply:
you only need one set of parentheses here. What I've suggested only removes the inner set.